src/js/control-bar/text-track-controls/chapters-button.js

```diff
@@ -78,7 +78,9 @@ class ChaptersButton extends TextTrackButton {
     let items = this.items = [];
 
     for (let i = 0, length = tracks.length; i < length; i++) {
-      let track = tracks[i];
+
+      // We will always choose the last track as our chaptersTrack
+      let track = tracks[length - 1];
```

Comment:
Actually, does this even need `createItems` since it's all happening in there? We should move the item creation login from this method into `createItems`.